getting-started/cloud-multi-node.md

```diff
@@ -0,0 +1,250 @@
+>:TIP:Timescale currently offers two options for cloud hosting. If you are a
+Timescale Forge user, please use the documentation for [setting up multi-node
+on Forge][forge-multi-node] instead.
+
+# Setting up TimescaleDB 2.0 Multi-node on Timescale Cloud
```

Comment:
One noteable difference for Multinode on Cloud is that a user can create a new database and setup a multinode for a specific database. Of course using existing is the most straight forward but if user wants he can create a new database on the AN and run all below commands (while adding data nodes they will automatically get provisioned with new database so no additional work there)

Reply:
After talking with @alexthinath , we decided to not mention this for now and decide if it's worth the effort later.  Thank you for remembering this and pointing it out.